src/start/RealmScreen.js

```diff
@@ -65,7 +65,7 @@ class RealmScreen extends PureComponent<Props, State> {
     const { progress, error, realm } = this.state;
 
     return (
-      <Screen title="Welcome" padding scrollView centerContent>
+      <Screen centerContent keyboardShouldPersistTaps="always" padding scrollView title="Welcome">
```

Comment:
I think alphabetization doesn't actually improve a list like this; the `title` is a pretty central bit of context when reading this code, so it's good that it's first.